packages/web-runtime/themes/owncloud/theme.json

```diff
@@ -1,8 +1,9 @@
 {
   "common": {
-      "name": "ownCloud",
-      "slogan": "ownCloud – A safe home for all your data",
-      "logo": "themes/owncloud/assets/logo.svg"
+    "name": "ownCloud",
+    "slogan": "ownCloud – A safe home for all your data",
+    "logo": "themes/owncloud/assets/logo.svg",
+    "loggedOutHelpUrl": "https://www.github.com"
```

Comment:
Is this used in production or just an example?

Reply:
Must be removed before merge. It appears in production instances otherwise. Probably just a dev leftover.

Reply:
yep will remove it after retro, and then get pipeline fixed